src/lib/mongoBackend/mongoSubCache.cpp

```diff
@@ -687,10 +101,10 @@ int mongoSubCacheItemInsert(const char* tenant, const BSONObj& sub)
   cSubP->throttling            = sub.hasField(CSUB_THROTTLING)?       getIntOrLongFieldAsLong(sub, CSUB_THROTTLING)       : -1;
   cSubP->expirationTime        = sub.hasField(CSUB_EXPIRATION)?       getIntOrLongFieldAsLong(sub, CSUB_EXPIRATION)       : 0;
   cSubP->lastNotificationTime  = sub.hasField(CSUB_LASTNOTIFICATION)? getIntOrLongFieldAsLong(sub, CSUB_LASTNOTIFICATION) : -1;
-  cSubP->count                 = sub.hasField(CSUB_COUNT)?            getIntOrLongFieldAsLong(sub, CSUB_COUNT)            : 0;
+  cSubP->count                 = 0;
```

Comment:
That is correct.

The problem I see it that we are "deep inside a function" when this *important* reset takes place.
It would be clearer and less error prone if we moved the entire 'count-reset', outside the re-creation of sub-cache-items.   Clearer and ess error-prone but also slower, of course, as we'd have to go through the entire list of subscriptions again ...

I just want your opinion on that.
Keep it where it is (a bit hidden but fast), or make it more visible but slower ...